docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
+This time the constraint violation is calculated using the :code:`ratio_bound`
+`0.9` and again with the definition from
+:ref:`constraints_binary_classification` and selection rate as the used
+metric:
+
+.. math::
+
+    r \cdot \text{metric}_a - \text{metric}_* = 0.9 \times 0.6 - 0.4 = 0.14 \\
+    r \cdot \text{metric}_b - \text{metric}_* = 0.9 \times 0.2 - 0.4 = -0.22 \\
+    - \text{metric}_a + r \cdot \text{metric}_* = - 0.6 + 0.9 \times 0.4 = -0.24 \\
+    - \text{metric}_b + r \cdot \text{metric}_* = - 0.2 + 0.9 \times 0.4 = 0.16 \\
+
+.. _true_positive_rate_parity:
+.. _false_positive_rate_parity:
+
+True Positive Rate Parity and False Positive Rate Parity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A classifier :math:`h(X)` satisfies *True Positive Rate Parity* if
+
+.. math::
+
+    \P[h(X) = 1 \given A = a, Y = 1] = \P[h(X) = 1 \given Y = 1] \; \forall a
+
+and *False Positive Rate Parity* if
+
+.. math::
+
+    \P[h(X) = 1 \given A = a, Y = 0] = \P[h(X) = 1 \given Y = 0] \; \forall a
+
+Note that this formulation of True Positive Rate Parity ignores what happens
+to samples with :math:`Y=0`, and the False Positive Rate Parity ignores
+:math:`Y=1`.
+Refer to :ref:`equalized_odds` for a constraint that extends this by
+considering all samples, i.e., the samples with :math:`Y=0` as well as the
+ones with :math:`Y=1`.
+
+In practice this can be used in a difference-based version as follows:
+
+.. doctest:: mitigation
+
+    >>> from fairlearn.reductions import TruePositiveRateParity
+    >>> from fairlearn.metrics import true_positive_rate_group_summary
+    >>> import numpy as np
+    >>> tprp = TruePositiveRateParity(difference_bound=0.01)
+    >>> X                  = np.array([[0], [1], [2], [3], [4], [5], [6], [7], [8], [9]])
+    >>> y_true             = np.array([ 1 ,  1 ,  1 ,  1 ,  1,   1 ,  1 ,  0 ,  0 ,  0 ])
+    >>> y_pred             = np.array([ 1 ,  1 ,  1 ,  1 ,  0,   0 ,  0 ,  1 ,  0 ,  0 ])
+    >>> sensitive_features = np.array(["a", "b", "a", "a", "b", "a", "b", "b", "a", "b"])
+    >>> true_positive_rate_group_summary(y_true, y_pred, sensitive_features=sensitive_features)
+    {'overall': 0.5714285714285714, 'by_group': {'a': 0.75, 'b': 0.3333333333333333}}
+    >>> tprp.load_data(X, y_true, sensitive_features=sensitive_features)
+    >>> tprp.gamma(lambda X: y_pred)
+    sign  event    group_id
+    +     label=1  a           0.178571
+                   b          -0.238095
+    -     label=1  a          -0.178571
+                   b           0.238095
+    dtype: float64
+
+.. note::
+
+    In mitigation examples we never need to call :code:`load_data` or
+    :code:`gamma` since the reductions techniques all automatically take care
+    of it. The example merely helps in illustrating the deviation from the
+    corresponding inequalities in :ref:`constraints_binary_classification`.
+
+Similarly to other constraints a ratio-based version is also available:
```

Comment:
Why isn't there an example like the others?

Reply:
There is, right below this line. Not sure what you're referring to.

Reply:
Hmmm... I'm wondering if I looked at the wrong doc build.